### ISSUE TITLE
Error in Inversion Task

### ISSUE THREAD

**Issue by ckorgial** [genforce/idinvert#52]
Hello !

could anyone help me find a solution for this Error?

InvalidArgumentError (see above for traceback): Default AvgPoolingOp only supports NHWC on device type CPU
[[node E_gpu0_1/Downscale2D/AvgPool (defined at <string>:244)  = AvgPool[T=DT_FLOAT, data_format="NCHW", ksize=[1, 1, 2, 2], padding="VALID", strides=[1, 1, 2, 2], _device="/job:localhost/replica:0/task:0/device:CPU:0"](E_gpu0_1/FromImg/batch_norm/batchnorm/add_1)]]

I have already installed the correct libraries and the correct python version. I also installed the pretrained model face_256x256 from the Google Drive. 

I run this command

python invert.py styleganinv_face_256.pkl examples/test.list

**Comment by zhujiapeng** [comment]
You need to run those scripts on GPU. Make sure you have installed the TensorFlow with GPU.

**Comment by ckorgial** [comment]
Thank you so much for your response !! I have solved my issue by installing tensorflow-gpu !!
